docs/limitations.md

```diff
@@ -1,12 +1,30 @@
 # Current limitations
 
-1. Does not work on Windows. Tested on Mac and Linux only. Tracking issue: https://github.com/prisma/prisma2/issues/4
-2. Processes 1 request at a time, no parallelism during the early preview period. This will be fixed before GA. 
-3. Some edge cases for complex nested mutations don't work properly. 
-4. Limited auto-complete in Typescript projects due to a compiler bug. We raised a PR to fix that [here](https://github.com/microsoft/TypeScript/pull/32100). 
-5. No realtime API/subscriptions. In the future, we will have Prisma events engine (after the GA) that would pave way for such a feature, there is no ETA for this yet. 
-6. Models must have an `@id` attribute and it must take one of these forms:
+During the Preview period of Prisma 2 there are missing features, limited performance and stability issues you should be aware of:
+
+## Functionality limitations
+
+- Embedded types are not implemented yet ([tracking issue](https://github.com/prisma/lift/issues/43))
+- Many other types are not fully implemented yet
+- Models must have an `@id` attribute and it must take one of these forms:
     - `Int @id`
     - `String @id @default(uuid())`
-    - `String @id @default(cuid())`
-7. When [introspecting](./introspection.md) a database, Prisma only recognizes many-to-many relations that follow the Prisma conventions for [relation tables](https://github.com/prisma/prisma2/blob/master/docs/relations.md#mn).
+    - `String @id @default(cuid())`  
+- When [introspecting](./introspection.md) a database, Prisma for now only recognizes many-to-many relations that follow the Prisma conventions for [relation tables](https://github.com/prisma/prisma2/blob/master/docs/relations.md#mn).
+- Some edge cases for complex nested mutations don't work properly yet.
```

Comment:
this should be "nested write" not "nested mutation"

Reply:
Ok, that was just copy/paste from the original thing. Was something implemented in the meantime that changes this?

Reply:
This is now tracked in https://github.com/prisma/prisma2/issues/602